extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpConfiguration.java

```diff
@@ -245,6 +245,12 @@
     @ConfigItem
     public Optional<PayloadHint> unhandledErrorContentTypeDefault;
 
+    /**
+     * Additional HTTP Headers always sent in the response
+     */
+    @ConfigItem
+    public Map<String, String> header;
```

Comment:
Should it be `headers`?

Reply:
It was `headers` previously, but then I realized that the config would look better in singular form:
Eg.
```properties
quarkus.htrp.header."Pragma"=no-cache
```